### ISSUE TITLE
Audit arabica genesis.json

### ISSUE THREAD

**Issue by rootulp** [celestiaorg/networks#176]
## Motivation

https://github.com/celestiaorg/celestia-app/issues/1363 was caused by arabica-4's `genesis.json` not containing an expected parameter. If we recreate a `genesis.json` via:

```shell
celestia-appd init $CHAINID --chain-id $CHAINID
```

then the genesis file will contain all the parameters expected by celestia-app. However, for arabica-4 and arabica-5 we've manually copied + pasted the genesis.json to retain a list of account addresses that we fund at genesis time (i.e. a faucet address).

## Problem

If we continue copy + pasting the genesis files, then we will fail to pick up on changes to default parameters or new parameters.

## Proposal

1. Audit the diff between arabica-5's genesis.json and a freshly created genesis.json
2. Before we start a new chain, have the gensis.json in version control and audited by celestia-app team

**Comment by rootulp** [comment]
Closing as won't do
